hack/spi/remote-secret-init.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# !!! Note that this script should not be used for production purposes !!!
+
+source $(dirname "$0")/utils.sh
+
+REMOTE_SECRET_APP_ROLE_FILE="$(realpath .tmp)/approle_remote_secret.yaml"
+
+function auth() {
+	if ! vaultExec "vault auth list | grep -q approle"; then
+		echo "setup approle authentication ..."
+		vaultExec "vault auth enable approle"
+	fi
+
+	if [ ! -d ".tmp" ]; then mkdir -p .tmp; fi
```

Comment:
You can use just `mkdir -p .tmp`. It will create the directory or just succeed (without changing the content) if the directory exists.